odoo/addons/test_lint/tests/test_checkers.py

```diff
@@ -522,3 +522,66 @@ def test_gettext_repr(self):
         self.assertEqual(len(errors), 2)
         for error in errors:
             self.assertEqual(error["symbol"], "gettext-repr")
+
+    def test_missing_gettext(self):
+        """Test that missing gettext call raises 'missing-gettext'."""
+        node = """
+            from odoo.exceptions import UserError
+            def some_function():
+                raise UserError('This is not translated')
```

Comment:
no need for import and the whole shebang
```suggestion
            UserError(_('This is translated'))
```